src/azul/__init__.py

```diff
@@ -1226,6 +1226,16 @@ class GitStatus(TypedDict):
     def _git_status_env(self) -> dict[str, str]:
         return {'azul_git_' + k: str(v) for k, v in self.git_status.items()}
 
+    @property
+    def _deployment_env(self) -> dict[str, str]:
+        # This variable won't be exported to the Lambda functions because it is
+        # lowercase. The variable is needed for :meth:`Config.Deployment.is_shared`,
```

Comment:
```suggestion
        # This variable won't be exported automatically to the Lambda functions because it is
        # lowercase. The variable is needed for :meth:`Config.Deployment.is_shared`,
```